### DISCUSSION TITLE
size of the models are not real in webxr

### DISCUSSION THREAD

**Discussion by zjafarpishe** [google/model-viewer#4393] [General]
In ar-modes="webxr scene-viewer quick-look" size of the models are not real and it is difficult to recognize the surface
device is android

**Comment by elalish** [comment]
I'm guessing you're referring to how the model is placed before ARCore has found the floor. If you move the phone around a bit (like you're required to do in QuickLook/SceneViewer) it will move up to proper placement. I would recommend adding some UX to help your users here, e.g. https://modelviewer.dev/examples/augmentedreality/#webXR
